code/game/machinery/lightswitch.dm

```diff
@@ -34,8 +34,12 @@
 /obj/machinery/light_switch/examine(mob/user)
 	. = ..()
 	. += "It is [area.lightswitch ? "on" : "off"]."
+	if(obj_flags & EMAGGED)
+		. += "You hear a faint hum coming from the switch."
```

Comment:
What if I'm deaf?

Reply:
then its your problem now innit

Reply:
NICH !!!! CAN YOU HEAR ME!??!  I WEEENT AAAHEEEAD AND COMMITED MY CHANGE
------